ocs_ci/ocs/perftests.py

```diff
@@ -221,8 +244,48 @@ def wait_for_wl_to_finish(self, timeout=18000, sleep=300):
             timeout (int): time in second to wait until the benchmark start
             sleep (int): Sleep interval seconds
 
+        Raise:
+            exception for too much restarts of the test.
+
         """
         log.info(f"Waiting for {self.client_pod_name} to complete")
+
+        Finished = 0
+        restarts = 0
+        while not Finished:
+            results = run_oc_command(
+                "get pod --no-headers -o custom-columns=:metadata.name,:status.phase",
+                namespace="my-ripsaw",
+            )
+            fname = ""
+            for name in results:
+                if re.search(self.client_pod_name, name):
+                    (fname, status) = name.split()
+                    continue
+            if not fname == self.client_pod:
+                log.info(
+                    f"The pod {self.client_pod} was restart. the new client pod is {fname}"
+                )
+                self.client_pod = fname
+                restarts += 1
+            if restarts > 5:
```

Comment:
imo, 3 restarts should be good enough threshold

Reply:
i will changed it.